packages/graphql-tool-utilities/config.ts

```diff
@@ -26,9 +26,7 @@ export function getGraphQLProjects(config: GraphQLConfig) {
 
 export function getGraphQLSchemaPaths(config: GraphQLConfig) {
   return getGraphQLProjects(config).reduce<string[]>((schemas, project) => {
-    return schemas.concat(
-      getGraphQLFilePath(config, project.resolveSchemaPath()),
-    );
+    return schemas.concat(project.resolveSchemaPath());
```

Comment:
`resolveSchemaPath` now performs the full path resolution, so we don't need to use `getGraphQLFilePath` any longer (i will eliminate `getGraphQLFilePath` in a new PR shortly).

Reply:
#38